src/dev/precommit_hook/casing_check_config.js

```diff
@@ -32,6 +32,7 @@ export const IGNORE_FILE_GLOBS = [
   '**/__mocks__/**/*',
   'x-pack/docs/**/*',
   'src/core/server/core_app/assets/fonts/**/*',
+  'packages/core/apps/core-apps-server-internal/src/assets/fonts/**/*',
```

Comment:
Pretty sure we can remove the old path, right?
```suggestion
  'packages/core/apps/core-apps-server-internal/src/assets/fonts/**/*',
```

Reply:
yea, agreed. there were a bunch of failing tests when I first tried to remove the old path but those turned out to be unrelated to this PR.